### PR TITLE
Update README.md with current skills and options

### PR DESCRIPTION
**Pull request by hosiawak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -201,3 +207,12 @@ resume normal operation.
 In any case, `clboss-ignore-onchain` is temporary and even
 if you forget to issue `clboss-notice-onchain` CLBOSS will
 resume managing onchain funds at some point.
+
+
+### Configuration
```
Comment (ZmnSCPxj) [inline README.md]: The `clboss-min-onchain` pull request already includes a description of `clboss-min-onchain` in the `README.md`, but I think separating the configuration options like you did here makes more sense (the `clboss-min-onchain` PR adds it next to the various commands).  That probably makes this PR conflict with the already-merged `clboss-min-onchain` PR, so maybe you can copy some of the text there to here?

File: README.md [hunk 2]
```diff
@@ -20,6 +20,12 @@ language is unusual today.
 I hope CLBOSS can make the transition from pre-Lightning to
 post-Lightning much smoother in practice.
 
+So far CLBOSS can do the following automatically:
+
+* Open channels to other, useful nodes when fees are low and there are onchain funds
+* Rebalance open channels using boltz.exchange swaps and forwards (JIT Rebalancer)
```
Comment (ZmnSCPxj) [inline README.md]: More exactly:

```markdown
* Acquire incoming capacity via `boltz.exchange` swaps.
* Rebalance open channels by self-payment (including JIT rebalancer).
```

**Review by ZmnSCPxj — COMMENTED** [review]
Thank you very much!

**Comment by hosiawak** [comment]
No worries, I'll fix it.

**Comment by hosiawak** [comment]
@ZmnSCPxj I removed the separate Configuration section because some of the options were commands above and this was the sole option.

**Pull request merged**